### PR TITLE
tikv-ctl.rs: fix command help message

### PR DESCRIPTION
**Pull request by ethan-daocloud**
the misspelled words maybe confusing, it `encoded` or `unencoded` ?

Signed-off-by: Guangming Wang <guangming.wang@daocloud.io>

## What have you changed? (mandatory)

fix typos words in help message

Please **NOTE** that:
- Do not assume reviewers understand the original issue

## What are the type of changes? (mandatory)

- Misc (other changes)

## How has this PR been tested? (mandatory)
local test.

## Does this PR affect documentation (docs) or release note? (mandatory)

NO

## Does this PR affect `tidb-ansible` update? (mandatory)

NO

### CODE REVIEW

File: cmd/src/bin/tikv-ctl.rs [hunk 1]
```diff
@@ -1639,7 +1639,7 @@ fn main() {
                         .short("k")
                         .required(true)
                         .takes_value(true)
-                        .help("The key to split it, in unecoded escaped format")
+                        .help("The key to split it, in encoded escaped format")
```
Comment (siddontang) [inline cmd/src/bin/tikv-ctl.rs]: PTAL @Connor1996

Comment (Connor1996) [reply cmd/src/bin/tikv-ctl.rs]: It should be `unencoded`

Comment (lonng) [reply cmd/src/bin/tikv-ctl.rs]: `uncoded`?

Comment (ethan-daocloud) [reply cmd/src/bin/tikv-ctl.rs]: got it, let's fix it

**Pull request merged**